allennlp/models/encoder_decoders/wikitables_semantic_parser.py

```diff
@@ -269,6 +339,57 @@ def forward(self,  # type: ignore
             # TODO(matt): compute accuracy here.
             return outputs
 
+    def _compute_average_with_encoder(self, embed_tensor: Variable,
```

Comment:
As I noted above, I think this function can be removed once you use `TimeDistributed`, which already implements what you did here.